src/lib/azure/storage.ts

```diff
@@ -149,7 +149,7 @@ export const getStorageAccountKeys = async (
     accountName
   );
 
-  if (typeof keysResponse.keys !== undefined) {
+  if (typeof keysResponse.keys !== "undefined") {
```

Comment:
Did you try `!typeof keysResponse.keys` option?

Reply:
sync'ed offline. typeof returns a string, therefore the comparison needs to be to "undefined" not just undefined.